src/browser-runner/platform-dummy/index.js

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2016 Mozilla
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+*/
+
+import colors from 'colors/safe';
+
+import logger from '../logger';
+
+import { serve } from '../serve';
+import { configureStore } from '../../shared/store/configure';
+import * as Endpoints from '../constants/endpoints';
+import Client from '../../shared/util/client';
+import RootSaga from './sagas/root-saga';
+import SharedActions from '../../shared/actions/shared-actions';
+
+const store = configureStore({ sagas: RootSaga });
+const client = new Client({ endpoint: Endpoints.WS_ROUTE_PROMISE, store, logger });
+
+if (!Endpoints.USING_EXTERNAL_SERVER) {
+  serve();
+}
```

Comment:
Not sure if this is at all necessary for the dummy runner, I'll send you a PR removing the --use-external-server nonsense from these browser runners. After all, in the case of the dummy runner, the build task is starting both the server and the runner. Only production packaged runner builds want to start a server.